module-2/lab-df-calculation-and-transformation/your-code/challenge-2.ipynb

```diff
@@ -13,7 +13,7 @@
   },
```

Comment:
- Use employee_by_department instead of calling numeric_cols if you want to get printed values per department.